Azure/azure-key-vault/_meta/fields.yml

```diff
@@ -0,0 +1,24 @@
+azure.key_vault.correlation_id:
```

Comment:
Could you create custom fields for the fields in identity.claim and the fields in properties ?

Reply:
@vg-svitla
Please add `identity.claim.scp` and` identity.claim.amr `
There is also more fields in properties such as the `id`, fields in `keyProperties`, `certificateProperties`, `secretProperties`, `isAccessPolicyMatch`, `tenantId`, etc. Please check `VaultPatch`, `CertificateImport (OK)` and `CertificateCreate` events.

Reply:
Ok seems like I did not understand the description of task in correct way. Will fix it